base/server/python/pki/server/cli/acme.py

```diff
@@ -718,8 +734,26 @@ def execute(self, argv):
         database_conf = os.path.join(acme_conf_dir, 'database.conf')
         config = {}
 
-        logger.info('Loading %s', database_conf)
-        pki.util.load_properties(database_conf, config)
+        if database_type:
+            source = '/usr/share/pki/acme/database/%s/database.conf' % database_type
```

Comment:
% is a printf style formatter used in python2. Since we are targetting python3+, we should try using .format()

Ref: https://stackoverflow.com/q/12382719

Reply:
Fixed.